giraffe/src/components/GeoLayer.tsx

```diff
@@ -36,6 +36,7 @@ interface LastRenderProperties {
   zoomOnLastRender?: number
   widthOnLastRender?: number
   heightOnLastRender?: number
+  delayedQueryTimerHandle?
```

Comment:
❓ : Should there be a type on this handle?

Reply:
The problem here is that the typescript runtime target is set to Node.js in giraffe where setTimeout returns NODEJS.Timer, but in browser it returns a number. If I would declare the type correctly, it wouldn't compile. Therefore I left this declaration as is. Seems to be a common problem as I googled.

Reply:
Later I figured I can declare it as ReturnType<typeof setTimeout>.